api/src/test/java/org/openmrs/api/OrderServiceTest.java

```diff
@@ -4101,4 +4137,19 @@ public void getOrderAttributeByUuid_shouldReturnOrderAttributeUsingProvidedUuid(
 		assertEquals("Testing Reference", orderAttribute.getValueReference());
 		assertEquals(1, orderAttribute.getId());
 	}
+
+	@Test
+	public void getOrderAttributeTypeByName_shouldReturnCorrectOrderAttributeType() {
+		String orderAttributeTypeName = "Supplies";
```

Comment:
https://wiki.openmrs.org/display/docs/Java+Conventions#JavaConventions-NamingConventions

Reply:
Added final and capitalized